src/data/faq.json

```diff
@@ -59,6 +59,22 @@
                             "Notes:<ul><li>The error reports are automatically deleted after 14 days</li><li>Error reports that are transmitted to the server without request by the technical support cannot be analyzed and processed</li><li>There is no direct feedback channel for the error reports in the Corona-Warn-App. Technical support can only communicate with you via one of the above mentioned channels</li></ul>",
                             "For more info as well as screenshots, see the blog article <a href='../blog/2021-05-12-corona-warn-app-version-2-2/' target='_blank' rel='noopener noreferrer'>Corona-Warn-App version 2.2 enables creation of a rapid test profile and error reports</a>"
                         ]
+                    },
+                    {
+                        "title": "Where can I get tested?",
+                        "anchor": "where_can_i_get_tested",
+                        "active": true,
+                        "textblock": [
+                            "<b>Rapid antigen test sites</b> connected to the Corona-Warn-App are offered by the following partners:",
+                            "<ul><li>Appointments can be booked in advance via their websites:",
+                            "<ul><li><a href='https://meintest.brk.de' target='_blank' rel='noopener noreferrer'>Bavarian Red Cross</a></li><li><a href='https://corona-schnelltest-zentren.dm.de' target='_blank' rel='noopener noreferrer'>dm-drogerie markt</a></li><li><a href='https://buergertest.ecocare.center' target='_blank' rel='noopener noreferrer'> EcoCare (Kaufland und Lidl)</a></li><li><a href='https://testbuchen.de' target='_blank' rel='noopener noreferrer'>No-Q (participating pharmacies, family doctors and other test centers)</a></li></ul>",
+                            "</li><li>Without central website for booking appointments:",
+                            "<ul><li>Doctorbox</li><li>Huber Health Care</li></ul>",
+                            "</li><li>Appointments only for their own employees:",
+                            "<ul><li>The Federal Chancellery tests its own employees</li><li>Saarland's Ministry of Education and Culture tests teachers</li></ul>",
+                            "</li></ul><hr>An overview of the <b>PCR and rapid antigen test sites</b> can be found on this <a ref=https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG webpage</a> (not clear whether the test site is connected to the CWA or not).", 

```

Comment:
This hyperlink doesn't work.
Did you intend to publish the comment "(not clear whether the test site is connected to the CWA or not)"?

Reply:
1. hyperlink. thanks. will fix
2. (not clear whether the test site is connected to the CWA or not) -> yes, should be in the text